test/unit/adb-emu-commands-specs.js

```diff
@@ -0,0 +1,90 @@
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import ADB from '../..';
+import { withMocks } from 'appium-test-support';
+import log from '../../lib/logger.js';
+
+chai.use(chaiAsPromised);
+chai.should();
+const emulators = [
+  { udid: 'emulator-5554', state: 'device', port: 5554 },
+  { udid: 'emulator-5556', state: 'device', port: 5556 }, 
+];
+const fingerprint = 1111;
+
+describe('adb emulator commands', () => {
+  let adb = new ADB();
+  describe("emu", () => {
+    describe("isEmulatorConnected", withMocks({adb}, (mocks) => {
+      it("should check emulator is connected", async () => {
+        mocks.adb.expects("getConnectedEmulators")
+          .atLeast(1).withExactArgs()
+          .returns(emulators);
+        (await adb.isEmulatorConnected("emulator-5554")).should.equal(true);
+        (await adb.isEmulatorConnected("emulator-5556")).should.equal(true);
+        (await adb.isEmulatorConnected("emulator-5558")).should.equal(false);
+        mocks.adb.verify();
+      });
+    }));
+    describe("fingerprint", withMocks({adb, log}, (mocks) => {
+      it("should emit fingerprint without error", async () => {
+        mocks.adb.expects("getConnectedEmulators")
+          .atLeast(1).withExactArgs()
+          .returns(emulators);
+        mocks.adb.expects("getApiLevel")
+          .atLeast(1).withExactArgs()
+          .returns("23");
+        mocks.adb.expects("setDeviceId")
+          .atLeast(1).withExactArgs("emulator-5554")
+          .returns();
+        mocks.adb.expects("adbExec")
+          .atLeast(1)
+          .withExactArgs(["emu", "finger", "touch", fingerprint])
+          .returns("");
+        chai.expect(await adb.fingerprint(fingerprint)).to.be.true;
```

Comment:
done it. 
https://github.com/vrunoa/appium-adb/blob/master/test/unit/adb-emu-commands-specs.js#L50